src/core/Akka.TestKit/TestKitBase_Expect.cs

```diff
@@ -75,7 +73,7 @@ public abstract partial class TestKitBase
             CancellationToken cancellationToken = default)
         {
             return ExpectMsgAsync(message, timeout, hint, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.